docs/README.android

```diff
@@ -188,42 +190,42 @@ on with the Android toolchain and creating an Android Application Package
 5.1. Building dependencies
 --------------------------------------------------------------------
 
-   # cd $HOME/kodi-android/tools/depends
-   # ./bootstrap
-   # ./configure --help
+    $ cd $HOME/kodi-android/tools/depends
+    $ ./bootstrap
+    $ ./configure --help
 
    Run configure with the correct settings for you local configuration.
    See tools/depends/README for examples.
 
    Anyone working on the dependencies themselves will want to set the
    environment variables specified in ~/.bashrc or similar, to avoid
-   having to input these with each configure. 
+   having to input these with each configure.
 
-   # make -j <jobs>
-   # make -j <jobs> -C target/binary-addons
+    $ make -j <jobs>
+    $ make -j <jobs> -C target/binary-addons
 
    NOTE: if you only want to build specific addons you can specify like this:
-   # make -C target/binary-addons ADDONS="pvr.hts pvr.dvblink"
+    $ make -C target/binary-addons ADDONS="pvr.hts pvr.dvblink"
 
    This build was designed to be massively parallel. Don't be afraid to
    give it a 'make -j20' or so.
 
    Verify that all deps built correctly (it will tell you so) before
    continuing. You will get crazy build errors otherwise. If in doubt
-   run another 'make' (single threaded) until the message 
+   run another 'make' (single threaded) until the message
    "Dependencies built successfully." appears. If the single make
-   fails keep cleaning the dependend library in charge by issueing
-   make -C target/<name_of_failed_lib> distclean. Then try make again.
+   fails keep cleaning the dependes library in charge by issuing
```

Comment:
Dont talk shit.
before it was dependend, which makes me think that "dependent" is meant.
you changed it to "dependes". That isnt even a regular word.